govc/vm/guest/auth.go

```diff
@@ -56,16 +56,20 @@ func (flag *AuthFlag) Register(ctx context.Context, f *flag.FlagSet) {
 	value := os.Getenv(env)
 	err := flag.Set(value)
 	if err != nil {
-		fmt.Printf("couldn't set guest login values: %v", err)
+		fmt.Printf("could not set guest login values: %v", err)
 	}
-	usage := fmt.Sprintf("Guest VM credentials [%s]", env)
+	usage := fmt.Sprintf("Guest VM credentials (<user>:<password>) [%s]", env)
 	f.Var(flag, "l", usage)
 	if flag.proc {
 		f.BoolVar(&flag.auth.GuestAuthentication.InteractiveSession, "i", false, "Interactive session")
 	}
 }
 
 func (flag *AuthFlag) Process(ctx context.Context) error {
+	if flag.auth.Username == "" || flag.auth.Password == "" {
```

Comment:
See also: https://github.com/vmware/govmomi/issues/2689#issuecomment-1007782485

Just having non-empty Username is good enough. It is possible for an OS user to have an empty password (tho unlikely these days)

Reply:
Ah, gotcha! Will fix